Core/HLE/sceFont.cpp

```diff
@@ -613,7 +618,7 @@ class FontLib {
 
 		u32 allocSize = 12;
 		if (mode == FONT_OPEN_INTERNAL_STINGY) {
-			allocSize = 0x239B4;
+			allocSize = loadedFont->GetFont()->getStingySize();
```

Comment:
a little curious that this one is = and the other one +=. Maybe the bias of 12 is builtin to the non-stingy sizes?